src/patternfly/components/AppLauncher/app-launcher.scss

```diff
@@ -166,6 +205,44 @@
       }
     }
   }
+
+  &.pf-m-wrapper {
+    --pf-c-app-launcher__menu-item--PaddingTop: var(--pf-c-app-launcher__menu-item--m-wrapper--PaddingTop);
+    --pf-c-app-launcher__menu-item--PaddingRight: var(--pf-c-app-launcher__menu-item--m-wrapper--PaddingRight);
+    --pf-c-app-launcher__menu-item--PaddingBottom: var(--pf-c-app-launcher__menu-item--m-wrapper--PaddingBottom);
+    --pf-c-app-launcher__menu-item--PaddingLeft: var(--pf-c-app-launcher__menu-item--m-wrapper--PaddingLeft);
+
+    align-items: stretch;
+  }
+
+  &.pf-m-link {
+    --pf-c-app-launcher__menu-item--PaddingTop: var(--pf-c-app-launcher__menu-item--m-link--PaddingTop);
+    --pf-c-app-launcher__menu-item--PaddingRight: var(--pf-c-app-launcher__menu-item--m-link--PaddingRight);
+    --pf-c-app-launcher__menu-item--PaddingBottom: var(--pf-c-app-launcher__menu-item--m-link--PaddingBottom);
+    --pf-c-app-launcher__menu-item--PaddingLeft: var(--pf-c-app-launcher__menu-item--m-link--PaddingLeft);
+    --pf-c-app-launcher__menu-item--hover--BackgroundColor: var(--pf-c-app-launcher__menu-item--m-link--hover--BackgroundColor);
+  }
+
+  &.pf-m-action {
```

Comment:
I wonder if this should be a more specific modifier name? Maybe the default is `pf-m-favorite` and the active modifier is `pf-m-favorite-active` or `pf-m-favorited`? They seem like they should be more connected.

Reply:
The way I did it is an attempt to make a generic action button that is separate from the link in a menu item. The intent is that we could add more actions here and it would scale, then add unique modifiers (like `.pf-m-favorite`) for unique things that apply to that individual action.

That said, I can take a more specific approach and name these elements (the link, too) something specific to the favorite feature if you think that's better. 

Do you think these existing classes will scale in a generic way?

Reply:
ah ok, I see what you mean. That's smart, I like it! 👍